lib/ez/settings/backend/active_record.rb

```diff
@@ -0,0 +1,32 @@
+module Ez::Settings
+  module Backend
+    class ActiveRecord
+      def read
+        hash = {}
+        ActiveRecordsStore.all.each do |settings|
+          hash[settings.group] ||= {}
+          hash[settings.group].merge!(settings.key => settings.value)
+        end
+        hash.deep_symbolize_keys
+      end
+
+      def write(data)
+        group = data.keys[0]
+        pairs = data.values[0]
+        existing_settings = ActiveRecordsStore.where(group: group, key: pairs.keys)
+        records = pairs.map do |pair|
+          record(existing_settings, pair) || ActiveRecordsStore.new(group: group, key: pair[0], value: pair[1])
+        end
+        records.each(&:save!)
+      end
```

Comment:
The implementation here a bit odd, but this allows to reduce the amount of queries while gathering the existing records from database.
The original approach was:
```ruby
keys = data.values.first.keys
pairs.map do |key, value|
  setting = ActiveRecordsStore.find_or_initialize_by(group: group, key: key)
  setting.value = value
  setting.save
end
```